OpenRA.Mods.Common/Widgets/Logic/MapChooserLogic.cs

```diff
@@ -105,10 +118,13 @@ public class MapChooserLogic : ChromeLogic
 			widget.Get<ButtonWidget>("BUTTON_CANCEL").OnClick = canceling;
 
 			gameModeDropdown = widget.GetOrNull<DropDownButtonWidget>("GAMEMODE_FILTER");
+			orderByDropdown = widget.GetOrNull<DropDownButtonWidget>("ORDERBY");
```

Comment:
this could be moved into `SetupOrderByDropdown`

Reply:
I'm not quite sure we should do that. At the moment the field is readonly. I need to remove readonly to assign it in the function. Should I do that or should we leave it as is?

Reply:
it appears this variable is only use inside the `SetupOrderByDropdown` function. So you might as well move the whole variable there

Reply:
all right, that sounds good!